frontend/src/scenes/insights/insightLogic.ts

```diff
@@ -720,13 +679,6 @@ export const insightLogic = kea<insightLogicType>({
                     { ...insight, ...createdInsight, result: createdInsight.result || insight.result },
                     {}
                 )
-                if (values.syncWithUrl) {
-                    router.actions.replace(
-                        values.insightMode === ItemMode.Edit
-                            ? urls.insightEdit(createdInsight.short_id, values.filters)
-                            : urls.insightView(createdInsight.short_id, values.filters)
-                    )
-                }
             }
         },
         createAndRedirectToNewInsight: async ({ filters }, breakpoint) => {
```

Comment:
Looks like we don't need `createAndRedirectToNewInsight` anymore?

Reply:
removed